document/src/vespa/document/util/stringutil.cpp

```diff
@@ -90,18 +90,17 @@ const vespalib::string & StringUtil::escape(const vespalib::string & source, ves
                                        char delimiter)
 {
     size_t escapeCount(0);
-    for(size_t i(0), m(source.size()); i < m; i++) {
-        if (source[i] == delimiter) {
+    for (char i : source) {
```

Comment:
Consider using `c` instead of `i`, as in a few lines below.

Reply:
Fixed